spiped/dispatch.c

```diff
@@ -125,6 +127,8 @@ callback_gotconn(void * cookie, int s)
 
 	/* We have gained a connection. */
 	A->nconn += 1;
+	if (A->only_one)
```

Comment:
The tests aren't in spiped yet, and it's easy enough for me to delete the one that tests opening two connections and closing one.  I think the main reason you wanted me to add that test was so that we could check if memory from active connections was being de-allocated before quitting.